spec/view_changes_need_screenshots_spec.rb

```diff
@@ -49,60 +49,60 @@ module Danger
         @plugin = @dangerfile.view_changes_need_screenshots
       end
 
-      context 'iOS PR has view code changes in Swift files' do
+      context 'with an iOS PR having view code changes in Swift files' do
```

Comment:
This change and all the similar ones below about the convention of beginning `context` names using `when` or `with` kinda self-advocate for breaking down those contexts in 2 levels:
```ruby
context 'when checking an iOS PR' do
  context 'with View code changes in Swift files' do … end
  context 'with Button code changes in Swift files' do … end
  context 'with View code changes in ObjC files' do … end
  context 'with Button code changes in Swift files' do … end
end

context 'when checking an Android PR' do
  context 'with View code changes in Kotlin files' do … end
  context 'with Button code changes in Kotlin files' do … end
  context 'with View code changes in Java files' do … end
  context 'with Button code changes in Java files' do … end
end
```

Reply:
Nice one 👍